buildifier/buildifier.go

```diff
@@ -292,6 +292,13 @@ func processFiles(files []string, inputType, lint string, warningsList []string)
 		data []byte
 		err  error
 	}
+
+	files, err := expandDirectories(files)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "buildifier: %v\n", err)
+		os.Exit(1)
```

Comment:
style wise is it usually cleaner if only 'main' is allowed to os.Exit

here, return fmt.Errorf(...)
[changing the function signature to return error]

Reply:
Done (moved the block out of the processFiles function).